### ISSUE TITLE
about a model you used: v8_n.pt 

### ISSUE THREAD

**Issue by di-ma-dl** [jahongir7174/YOLOv8-human#9]
I have a simple question. v8_n.pt  is just renamed version of the original yolov8n.pt model, isn't it? Thank you, in advance!

**Comment by jahongir7174** [comment]
yes, right if it is trained on the coco dataset, but the weight file can be different based on the trained dataset
